upstash/manifest.json

```
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "b6204e82-0732-4ab4-bca6-ed2e727ea581",
+  "app_id": "upstash",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
```

This is needed to surface the uninstallation instructions from the `README` in the tile in the datadog app:
```suggestion
    "configuration": "README.md#Setup",
    "uninstallation": "README.md#Uninstallation",
```